frontend/packages/dev-console/src/components/import/import-submit-utils.ts

```diff
@@ -314,8 +394,19 @@ export const createResources = async (
     );
     return Promise.all([k8sCreate(KnServiceModel, knDeploymentResource)]);
   }
-
-  requests.push(createDeploymentConfig(formData, imageStream, dryRun));
+  if (formData.resources === Resources.Kubernetes) {
+    const imageStreamResponse = await requests.shift();
+    requests.push(
+      createDeployment(
+        formData,
+        imageStreamResponse.status.dockerImageRepository,
+        imageStream,
+        dryRun,
+      ),
+    );
+  } else {
```

Comment:
I'm starting to wonder if we should do better than just `else` here and [here](https://github.com/openshift/console/pull/3168/files#diff-108abe60978679a6b7525c74ef65f375R292)... I wonder if we should explicitly call out the `Resources.Openshift` key. Perhaps even use a switch case. I am not sure it's super obvious that this `else` means `Resources.Openshift`.

Reply:
done